CMakeLists.txt

```diff
@@ -372,6 +372,15 @@ if(APPLE)
 
   install (TARGETS devilutionx DESTINATION ./)
 
+  if(DIST)
+      set(CMAKE_INSTALL_PREFIX ${CMAKE_BINARY_DIR})

```

Comment:
Why not just use CMAKE_BINARY_DIR directly?

Reply:
Can be CMAKE_BINARY_DIR directly, I only separated it for testing purposes :sweat_smile: